perceval/converters/abstract_converter.py

```diff
@@ -77,15 +79,24 @@ def __init__(self, backend_name: str = "SLOS", source: Source = None, noise_mode
     def count_qubits(self, gate_circuit) -> int:
         pass
 
+    @abstractmethod
+    def _get_gate_sequence(self, gate_circuit) -> list[list]:
+        pass
```

Comment:
Could you write a small docstring instead of `pass`, explaining what's expected from the implemented methods?